.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jenkinsci/custom-jenkins-distribution-service-developers
```

Comment:
It's good to always has a new empty line at the end file.

Reply:
Yes adding :+1: